app/vendor/README.md

```diff
@@ -32,7 +32,7 @@ Modal windows
 ### [jQuery](https://github.com/jquery/jquery)
 
 **Author**: jQuery Foundation and other contributors
-**License**: MIT, https://github.com/jquery/jquery/blob/master/MIT-LICENSE.txt
```

Comment:
https://github.com/jquery/jquery/blob/master/LICENSE.txt is correct (no *MIT-*).